test/functional/uvm_scsi_test.go

```diff
@@ -28,22 +29,25 @@ func TestSCSIAddRemoveLCOW(t *testing.T) {
 	u := testutilities.CreateLCOWUVM(context.Background(), t, t.Name())
 	defer u.Close()
 
-	testSCSIAddRemove(t, u, `/run/gcs/c/0/scsi`, "linux", []string{})
+	testSCSIAddRemoveMultiple(t, u, `/run/gcs/c/0/scsi`, "linux", []string{})
 
 }
 
 // TestSCSIAddRemoveWCOW validates adding and removing SCSI disks
 // from a utility VM in both attach-only and with a container path.
 func TestSCSIAddRemoveWCOW(t *testing.T) {
 	testutilities.RequiresBuild(t, osversion.RS5)
-	u, layers, uvmScratchDir := testutilities.CreateWCOWUVM(context.Background(), t, t.Name(), "microsoft/nanoserver")
+	// TODO make the image configurable to the build we're testing on
+	u, layers, uvmScratchDir := testutilities.CreateWCOWUVM(context.Background(), t, t.Name(), "mcr.microsoft.com/windows/nanoserver:1903")
 	defer os.RemoveAll(uvmScratchDir)
 	defer u.Close()
 
-	testSCSIAddRemove(t, u, `c:\`, "windows", layers)
+	testSCSIAddRemoveSingle(t, u, `c:\`, "windows", layers)
 }
 
-func testSCSIAddRemove(t *testing.T, u *uvm.UtilityVM, pathPrefix string, operatingSystem string, wcowImageLayerFolders []string) {
+// TODO this test is only needed until WCOW supports adding the same scsi device to
+// multiple containers
+func testSCSIAddRemoveSingle(t *testing.T, u *uvm.UtilityVM, pathPrefix string, operatingSystem string, wcowImageLayerFolders []string) {
```

Comment:
There's a lot of duplication between this and the multiple function. Can we combine them and change behavior at a few points based on a parameter?

Reply:
yes, but I figured testSCSIAddRemoveSingle would be removed when WCOW supports re-adding scsi devices, so I don't feel like its necessary.

Reply:
Are you good with that?

Reply:
Please add a comment explaining when this can be removed, and why it isn't combined with the other function.

Reply:
There's already a comment about when it can be removed :)